lib/rewrite-amd-definition.js

```diff
@@ -26,20 +28,20 @@ function getDependenciesForDefine(node) {
   }
 }
 
-function isAMDFunctionBody(node) {
+function isAMDFunctionBody(node, deps) {
   return namedTypes.FunctionExpression.check(node) &&
     node.id === null &&
     !!node.params &&
+    node.params.length === deps.length &&
     namedTypes.Identifier.check(node.params[0]) &&
-    node.params[0].name === 'exports';
+    (node.params[0].name === 'exports' || node.params[0].name === inflect.camelize(inflect.underscore(deps[0]), false));
```

Comment:
Well, this is to make the import of d3-selection-multi working.
Actually, there should be a better way, but all I know is that when parsing the function body of d3-selection-multi, there is no exports at all, so without adding this check, the plugin is ignored, and the prototype of d3 `select` is not extended with the `styles`or `attrs` functions. see https://github.com/d3/d3-selection-multi/blob/master/index.js#L9

Also the inflection is because deps[0] is `d3-selection` whereas node.params[0].name is `d3Selection`.

To be honest I'm kind of 
![no idea](https://cdn-images-1.medium.com/max/600/1*snTXFElFuQLSFDnvZKJ6IA.png)